manifests/windows.pp

```diff
@@ -18,10 +18,24 @@
     provider => 'chocolatey',
     require  => Class['chocolatey'],
   }
-  package { 'devbox-unzip':
-    ensure   => present,
-    provider => 'chocolatey',
-    require  => Package['devbox-common.extension'],
+ 
+  # Unzip package is broken on chocolatey so download directly
+  exec { 'curl http://iweb.dl.sourceforge.net/project/gnuwin32/unzip/5.51-1/unzip-5.51-1.exe -Outfile C:\unzip.exe':
+    provider => powershell,
+    creates  => 'C:\unzip.exe',
```

Comment:
urk. Can we put it it `C:/Program Files` or some such? I hate polluting the root directory, and that should be in the path, right? Also, use forward slashes on Windows too.